src/angular.bind.js

```diff
@@ -1,6 +1,8 @@
 if (window.angular.bootstrap) {
   //AngularJS is already loaded, so we can return here...
-  console.log('WARNING: Tried to load angular more than once.');
+  if (!!window.console) {
```

Comment:
that's personal preference. I prefer to test if something is `true` rather than `truthy` ; `!!window.console` is a boolean while `window.console` is _something_

I don't think it has any impact on anything so if you prefer `if (window.console)` i'll change it